### ISSUE TITLE
OKHTTP_VERSION incompatibility with other plugins

### ISSUE THREAD

**Issue by ceaheart** [silkimen/cordova-plugin-advanced-http#268]
I am running into an incompatibility issue with the OKHTTP_VERSION.  There is another plugin that references OKHTTP_VERSION 3.12.+

My question is: does your plugin support this version, and if so, what is the proper way to change it?

**System info**
 - Plugin version - 2.2
 - Running on Android v9
 - Phone: OnePlus 5
-  Other
Ionic:

   Ionic CLI                     : 5.4.4 (C:\Users\Chris\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.6.2
   @angular-devkit/build-angular : 0.11.4
   @angular-devkit/schematics    : 7.1.4
   @angular/cli                  : 7.1.4
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   Cordova CLI       : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms : android 7.1.4, browser 5.0.4, ios 4.5.5
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 2.5.2, (and 10 other plugins)      

Utility:

   cordova-res : 0.7.0
   native-run  : 0.2.9

System:

   Android SDK Tools : 26.1.1 (C:\Users\Chris\AppData\Local\Android\sdk\)
   NodeJS            : v10.16.3 (C:\Program Files\nodejs\node.exe)
   npm               : 6.12.0
   OS                : Windows 10

**Are you using ionic-native-wrapper?**
No

**Comment by silkimen** [comment]
To be honest, I've not tested it with a newer version. You'll need to try it. You can provide a param during plugin installation e.g. `cordova plugin add cordova-plugin-advanced-http --variable OKHTTP_VERSION="3.12.0"`. Can you please provide feedback after having updated and tested it?

**Comment by planbsim** [comment]
I´m using the [uk.co.workingedge.phonegap.plugin.launchnavigator](https://github.com/dpa99c/phonegap-launch-navigator) which currently use OKHTTP Version 3.12.0. I want to avoid compability problems, so I have installed the cordova-plugin-advanced-http with OKHTTP version 13.12.0 as described above. Using 3.12.0 brings the [Build Error](https://github.com/square/okhttp/issues/4395). Updating to 3.12.1 solved the build error.

Now I´m running it with OKHTTP 3.12.1 and as far as I have tested it, it runs without any problems. I will keep you in the loop if any issues occur.
